ops/jenkins/bfd-data-npi/Jenkinsfile

```diff
@@ -151,10 +151,13 @@ mvn deploy "-DaltDeploymentRepository=bfd-mgmt-bfd-mgmt::default::https://bfd-mg
 '''
                                 }
                                 awsAuth.assumeRole()
-                                sh """
-aws ssm put-parameter --name /bfd/mgmt/versioning/nonsensitive/bfd-data-npi-sha256 \
---type String --region us-east-1  --overwrite --value "${npiOrgNameSha256}"
-"""
+                                awsSsm.putParameter(
+                                    parameterName: "/bfd/mgmt/versioning/nonsensitive/bfd-data-npi-sha256",
+                                    parameterValue: "${npiOrgNameSha256}",
+                                    parameterType: "String",
+                                    parameterTags: "Key=Source,Value=${JOB_NAME} Key=Environment,Value=mgmt Key=stack,Value=mgmt Key=Terraform,Value=False Key=application,Value=bfd Key=business,Value=oeda",
+                                    shouldOverwrite: true
+                                )
```

Comment:
Does this not need the region? (It was there before, and I see it added in another tag below)

Reply:
I moved this into the putParameter function. Looks like this. This way it can be passed in or it defaults to `us-east-1`.
`awsRegionOpt = "--region ${args.awsRegion ?: 'us-east-1'}"`